Makefile

```diff
@@ -30,7 +30,7 @@ license: FORCE
 	python scripts/update_headers.py
 
 format: license FORCE
-	black --extend-exclude=\.ipynb .
+	black --extend-exclude=\.ipynb --check . || black --exclude=\.ipynb --check .
```

Comment:
In the new version, there is no flag `--extend-exclude`. So I added this additional commend in case the former failed.

Reply:
I think it's better to pin black version than to use this workaround. The reason is that, if the first `black --extend-exclude...` legitimately fails with an important error message, that message will then be converted to an `unknown command line option` error message, confusing devs.